libs/components/validation/src/lib/modules/validators/validators.ts

```diff
@@ -32,15 +33,53 @@ export class SkyValidators {
    * immediately when users enter values.
    * @param control
    */
-  public static url(control: AbstractControl): ValidationErrors | null {
-    const value = control.value;
+  public static url(
+    abstractControl: AbstractControl
+  ): ValidationErrors | undefined;
```

Comment:
Since this is a public method, we'll need to keep the return type the same. If we absolutely need to include `undefined` as a return type, you could do `: ValidationErrors | null | undefined`, but we probably want to keep it as `null` if possible to avoid confusion.